modules/combined/tests/inelastic_strain/creep/creep_nl1.i

```diff
@@ -14,9 +14,9 @@
   displacements = 'disp_x disp_y'
 []
 
-[Mesh]#Comment
```

Comment:
Once upon a time, MOOSE wouldn't parse a comment that appeared in just that form.  I added this comment in an input file for testing, and it has been copied and pasted all over the place.